unit-tests/debug_protocol/test-build-command.py

```diff
@@ -33,6 +35,7 @@ def send_hardware_monitor_command(device, command):
 try:
     ctx = rs.context()
     dev = ctx.query_devices()[0]
+    expected_status = convert_bytes_to_decimal("10 00 00 00")

```

Comment:
The return status code should be the same as the opcode... let's make that clear somehow